zebra/rib.h

```diff
@@ -637,6 +637,7 @@ extern uint32_t rt_table_main_id;
 void route_entry_dump_nh(const struct route_entry *re, const char *straddr,
 			 const struct vrf *re_vrf,
 			 const struct nexthop *nexthop);
+extern void zebra_main_router_started(void);
```

Comment:
hmm - header file - maybe this should be in zebra_router.h? that's a little more general-purpose